report/assets/styles.css

```diff
@@ -874,6 +874,7 @@
 .lh-audit--load-opportunity .lh-audit__display-text {
   text-align: right;
   flex: 0 0 calc(4 * var(--report-font-size));
+  white-space: nowrap;
```

Comment:
If you look closely, the "after" sparkline no longer lines up at the end. This is the same result as just deleting the `flex: ` property above, but defeats the purpose of the rule: these sparklines need to line up across these isolated flex containers.

Reply:
Yeah that was intentional, in the original PR. Also I think if we use tables for rendering tables, this issue might not have occurred. 

I've updated the PR with 5 and removed nowrap.